spec/identity/index.html

```diff
@@ -631,7 +631,7 @@ <h3 id="h3_publishing-a-webid-profile-using-the-rdfa-html-notation"><span class=
 <div class="informative section" id="privacy">
 <h3 id="h3_privacy"><span class="secno">5.4 </span>Privacy</h3><p><em>This section is non-normative.</em></p>
 <p>
-A WebID Profile is a public document that may contain public as well personal information about the agent identified by the WebID.
+A WebID Profile Document may contain public as well personal information about the agent identified by the WebID.
```

Comment:
1. missing "as"
2. "personal" is only applicable to human agents, but WebIDs may also denote software (and other) agents; "private" is therefore a better adjective.
```suggestion
A WebID Profile Document may contain public as well as private information about the agent identified by the WebID.
```